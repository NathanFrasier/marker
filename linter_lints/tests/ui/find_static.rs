
static TEST: u32 = 4;


fn main() {}
