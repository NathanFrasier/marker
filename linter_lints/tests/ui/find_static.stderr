warning: hey there is a static item here
 --> $DIR/find_static.rs:2:1
  |
2 | static TEST: u32 = 4;
  | ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(linter::test_lint)]` on by default

warning: 1 warning emitted

